ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-main.yaml

```
@@ -162,15 +163,15 @@ tests:
           memory: 200Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-hypershift
-- as: e2e-ocp-4-17
+- as: e2e-ocp-4-18
   optional: true
   steps:
     cluster_profile: ossm-aws
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:next
       RELEASE_IMAGE_LATEST: release:next
     env:
-      MAISTRA_BUILDER_IMAGE: quay-proxy.ci.openshift.org/openshift/ci:ci_maistra-builder_upstream-master
+      MAISTRA_BUILDER_IMAGE: gcr.io/istio-testing/build-tools:master-3d91e3d29bd1057b14995647d90cbf85e043eba6
```

```suggestion
      MAISTRA_BUILDER_IMAGE: gcr.io/istio-testing/build-tools:master-3d91e3d29bd1057b14995647d90cbf85e043eba6
       HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.18
      
```

I haven't seen a 4.18 tag from other hypershift usage in CI. We may need to wait for hypershift supporting a 4.18 builder image.

---

we may try the `$RELEASE_IMAGE_LATEST`